examples/kyb-app/src/components/atoms/Stepper/components/atoms/Breadcrumb/hooks/useBreadcrumb.ts

```diff
@@ -0,0 +1,4 @@
+import { breadcrumbContext } from '@app/components/atoms/Stepper/components/atoms/Breadcrumb/breadcrumb.context';
+import { useContext } from 'react';
+
+export const useBreadcrumb = () => useContext(breadcrumbContext);
```

Comment:
Would also be consistent with other instances.
```suggestion
export const useBreadcrumbContext = () => useContext(breadcrumbContext);
```